### ISSUE TITLE
Missing types after ol 7 update

### ISSUE THREAD

**Issue by bozhidarc** [Siedlerchr/types-ol-ext#95]
Hi I found the following missing types after the release of OL 7.
I think those can be fixed by replacing them with `ReturnType<Geometry["getType"]>` and `ReturnType<Overlay["getPositioning"]`

```
Error: node_modules/@types/ol-ext/overlay/Popup.d.ts:84:33 - error TS2304: Cannot find name 'OverlayPositioning'.

84     setPositioning(pos?: typeof OverlayPositioning | string): void;
                                   ~~~~~~~~~~~~~~~~~~


Error: node_modules/@types/ol-ext/overlay/Tooltip.d.ts:5:32 - error TS2307: Cannot find module 'ol/OverlayPositioning' or its corresponding type declarations.

5 import OverlayPositioning from 'ol/OverlayPositioning';
                                 ~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by Siedlerchr** [comment]
Thanks. OverlayPositioninig as been renamed to Positioninig.
I also found some issues in the examples
https://github.com/Viglino/ol-ext/issues/855

**Comment by Siedlerchr** [comment]
WIP: https://github.com/Siedlerchr/types-ol-ext/pull/96
I currently still have troubles getting the examples up and running. If you have any idea

**Comment by Siedlerchr** [comment]
I just released a version 3.0.0 which is now compatible with ol.ext 4.0.1 and ol7
